darts/models/components/PowerNorm.py

```diff
@@ -0,0 +1,247 @@
+# File   : MaskPowerNorm.py
```

Comment:
Same here, could you rename to `power_norm.py` ? Actually I'd be inclined to put them all in a single `layer_norm_variants.py` file.